### ISSUE TITLE
MissingMethodException JetBrains.ReSharper.UnitTestFramework.Criteria.IUnitTestElementCriterion 

### ISSUE THREAD

**Issue by inyutin-maxim** [SpecFlowOSS/SpecFlow.Rider#159]
```
com.jetbrains.rdclient.util.BackendException: Failed to evaluate the data rule. Метод не найден: "JetBrains.ReSharper.UnitTestFramework.Criteria.IUnitTestElementCriterion JetBrains.ReSharper.UnitTestFramework.Criteria.DisjunctiveCriterion.From(System.Collections.Generic.IEnumerable`1<JetBrains.ReSharper.UnitTestFramework.Criteria.IUnitTestElementCriterion>)".

--- EXCEPTION #1/3 [MissingMethodException]
Message = “Метод не найден: "JetBrains.ReSharper.UnitTestFramework.Criteria.IUnitTestElementCriterion JetBrains.ReSharper.UnitTestFramework.Criteria.DisjunctiveCriterion.From(System.Collections.Generic.IEnumerable`1<JetBrains.ReSharper.UnitTestFramework.Criteria.IUnitTestElementCriterion>)".”
ExceptionPath = Root.InnerException.InnerException
ClassName = System.MissingMethodException
HResult = COR_E_MISSINGMETHOD=80131513
Source = ReSharperPlugin.SpecflowRiderPlugin
StackTraceString = “
  в ReSharperPlugin.SpecflowRiderPlugin.UnitTestExplorers.SpecFlowTestsDataRules.GetSpecFlowUnitTestElements(IDataContext context)
     в JetBrains.Application.DataContext.DataRule`1.JetBrains.Application.DataContext.IDataRule.Evaluate(IDataContext context)
     в JetBrains.Application.DataContext.DataContext.Evaluator.TryEvaluateDataRules(IDataConstant constant, IDataContext dataContext, DataContextValidator validator)
”

--- Outer ---

--- EXCEPTION #2/3 [InvalidOperationException]
Message = “Failed to evaluate the data rule.”
ExceptionPath = Root.InnerException
ClassName = System.InvalidOperationException
InnerException = “Exception #1 at Root.InnerException.InnerException”
HResult = COR_E_INVALIDOPERATION=80131509

--- Outer ---

--- EXCEPTION #3/3 [LoggerException]
Message = “Failed to evaluate the data rule.”
ExceptionPath = Root
ClassName = JetBrains.Util.LoggerException
InnerException = “Exception #2 at Root.InnerException”
HResult = COR_E_APPLICATION=80131600
StackTraceString = “
  в JetBrains.Util.LoggerBase.Log(LoggingLevel level, String message, Exception ex)
     в JetBrains.Util.Logging.Logger.LogException(Exception ex)
     в JetBrains.Application.DataContext.DataContext.Evaluator.TryEvaluateDataRules(IDataConstant constant, IDataContext dataContext, DataContextValidator validator)
     в JetBrains.Application.DataContext.DataContext.DoGetData(IDataConstant constant)
     в JetBrains.Application.DataContext.DataContext.JetBrains.Application.DataContext.IDataContext.GetData[T](DataConstant`1 dataConstant)
     в JetBrains.ReSharper.UnitTestFramework.Actions.UnitTestRunBaseAction.Update(IDataContext context, ActionPresentation presentation, DelegateUpdate nextUpdate)
     в JetBrains.Application.UI.ActionsRevised.Handlers.ExecutableActionEvaluator.Evaluate(IAction handler, EvaluatedAction result, IDataContext context)
     в JetBrains.ReSharper.Feature.Services.Actions.ServicesActionEvaluator.Evaluate(IAction handler, EvaluatedAction result, IDataContext context)
     в JetBrains.Application.UI.ActionsRevised.Handlers.BoundAction.Evaluate(IDataContext context)
     в JetBrains.Platform.RdFramework.Actions.Backend.RdActions.CreateEvaluatedActionDescriptor(IActionDefWithId actionDefWithId, IDataContext dataContext, IActionHandlers handlers)
     в JetBrains.Platform.RdFramework.Actions.Backend.RdActionsAsyncAvailability.UpdateAction(ActionUpdateRequestData req, IDataContext dataContext)
     в JetBrains.Platform.RdFramework.Actions.Backend.RdActionsAsyncAvailability.UpdateOnBackgroundThread(IPriorityQueue`1 actionsToUpdate)
     в JetBrains.Application.Threading.Tasks.TaskHost.AccessViolationCatcher(Action action)
     в JetBrains.Application.Threading.Tasks.TaskHost.<>c__DisplayClass36_0.<Create>b__1(Object state)
     в System.Threading.Tasks.Task.Execute()
     в System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
     в System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
     в System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
     в System.Threading.Tasks.Task.ExecuteEntry(Boolean bPreventDoubleExecution)
     в JetBrains.Application.Threading.Tasks.Scheduler.JetScheduler.ExecuteTask(Task task)
     в JetBrains.Application.Threading.Tasks.Scheduler.JetSchedulerThread.EnqueueNextTask()
     в JetBrains.Application.Threading.Tasks.Scheduler.JetSchedulerThread.ThreadPoolProc()
     в JetBrains.Util.Reflection.CallStackAnnotation.CatchAnnotatedInvocation[TClassOfNewFrame](String methodNameOfNewFrame, Action actionToAnnotate)
     в System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
     в System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
     в System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
     в System.Threading.ThreadHelper.ThreadStart()
”

	at com.jetbrains.rdclient.protocol.components.RdClientProtocolComponent$2.invoke(RdClientProtocolComponent.kt:49)
	at com.jetbrains.rdclient.protocol.components.RdClientProtocolComponent$2.invoke(RdClientProtocolComponent.kt:24)
	at com.jetbrains.rd.util.reactive.Signal.fire(Signal.kt:32)
	at com.jetbrains.rd.framework.impl.RdSignal.onWireReceived(RdSignal.kt:42)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:56)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:11)
	at com.jetbrains.rd.framework.impl.ProtocolContexts.readMessageContextAndInvoke(ProtocolContexts.kt:151)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:55)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:11)
	at com.jetbrains.rdclient.protocol.RdDispatcher.flushAll(RdDispatcher.kt:97)
	at com.jetbrains.rdclient.protocol.RdDispatcher.access$flushAll(RdDispatcher.kt:18)
	at com.jetbrains.rdclient.protocol.RdDispatcher$flushAllRunnable$1.run(RdDispatcher.kt:23)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:214)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:196)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:805)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:348)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:82)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:131)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:47)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:187)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:891)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:760)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:447)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:818)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:446)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:805)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:492)
	at com.intellij.ide.IdeEventQueue.pumpEventsForHierarchy(IdeEventQueue.java:951)
	at com.intellij.openapi.progress.util.ProgressWindow.lambda$startBlocking$6(ProgressWindow.java:221)
	at com.intellij.openapi.application.impl.ApplicationImpl.runUnlockingIntendedWrite(ApplicationImpl.java:831)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:217)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$execFromEDT$6(ProgressRunner.java:321)
	at java.base/java.util.concurrent.CompletableFuture.uniAcceptNow(CompletableFuture.java:753)
	at java.base/java.util.concurrent.CompletableFuture.uniAcceptStage(CompletableFuture.java:731)
	at java.base/java.util.concurrent.CompletableFuture.thenAccept(CompletableFuture.java:2108)
	at com.intellij.openapi.progress.impl.ProgressRunner.execFromEDT(ProgressRunner.java:318)
	at com.intellij.openapi.progress.impl.ProgressRunner.submit(ProgressRunner.java:259)
	at com.intellij.openapi.progress.impl.ProgressRunner.submitAndGet(ProgressRunner.java:185)
	at com.intellij.openapi.application.impl.ApplicationImpl.runProcessWithProgressSynchronously(ApplicationImpl.java:424)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:553)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcessWithProgressSynchronously(ProgressManagerImpl.java:91)
	at com.intellij.openapi.progress.impl.CoreProgressManager.run(CoreProgressManager.java:389)
	at com.intellij.openapi.progress.Task.queue(Task.java:124)
	at org.sonarlint.intellij.analysis.AnalysisManager.submitManual(AnalysisManager.java:80)
	at org.sonarlint.intellij.trigger.SonarLintSubmitter.submitFilesModal(SonarLintSubmitter.java:77)
	at org.sonarlint.intellij.trigger.SonarLintCheckinHandler.beforeCheckin(SonarLintCheckinHandler.java:98)
	at com.intellij.vcs.commit.NonModalCommitWorkflow.runBeforeCommitHandler(NonModalCommitWorkflow.kt:32)
	at com.intellij.vcs.commit.AbstractCommitWorkflow.runBeforeCommitHandlersChecks(AbstractCommitWorkflow.kt:243)
	at com.intellij.vcs.commit.NonModalCommitWorkflow.runHandlers(NonModalCommitWorkflow.kt:80)
	at com.intellij.vcs.commit.NonModalCommitWorkflowHandler.runAllHandlers(NonModalCommitWorkflowHandler.kt:191)
	at com.intellij.vcs.commit.NonModalCommitWorkflowHandler.access$runAllHandlers(NonModalCommitWorkflowHandler.kt:41)
	at com.intellij.vcs.commit.NonModalCommitWorkflowHandler$doExecuteDefault$1$1.invokeSuspend(NonModalCommitWorkflowHandler.kt:172)
	at com.intellij.vcs.commit.NonModalCommitWorkflowHandler$doExecuteDefault$1$1.invoke(NonModalCommitWorkflowHandler.kt)
	at com.intellij.vcs.commit.NonModalCommitWorkflow.checkCommit(NonModalCommitWorkflow.kt:51)
	at com.intellij.vcs.commit.NonModalCommitWorkflow.executeDefault(NonModalCommitWorkflow.kt:38)
	at com.intellij.vcs.commit.NonModalCommitWorkflowHandler$doExecuteDefault$1.invokeSuspend(NonModalCommitWorkflowHandler.kt:161)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.intellij.openapi.application.constraints.BaseConstrainedExecution$Companion$scheduleWithinConstraints$1.invoke(BaseConstrainedExecution.kt:68)
	at com.intellij.openapi.application.constraints.BaseConstrainedExecution$Companion.scheduleWithinConstraints(BaseConstrainedExecution.kt:71)
	at com.intellij.openapi.application.constraints.BaseConstrainedExecution.scheduleWithinConstraints(BaseConstrainedExecution.kt:38)
	at com.intellij.openapi.application.impl.BaseExpirableExecutorMixinImpl.access$scheduleWithinConstraints$s1153900543(BaseExpirableExecutorMixinImpl.kt:12)
	at com.intellij.openapi.application.impl.BaseExpirableExecutorMixinImpl$scheduleWithinConstraints$$inlined$Runnable$1.run(Runnable.kt:19)
	at com.intellij.openapi.application.impl.AppUIExecutorImpl$MyEdtExecutor.execute(AppUIExecutorImpl.kt:72)
	at com.intellij.openapi.application.impl.BaseExpirableExecutorMixinImpl.scheduleWithinConstraints(BaseExpirableExecutorMixinImpl.kt:23)
	at com.intellij.openapi.application.constraints.ConstrainedCoroutineDispatcherImpl.dispatch(ConstrainedCoroutineDispatcher.kt:34)
	at kotlinx.coroutines.internal.DispatchedContinuationKt.resumeCancellableWith(DispatchedContinuation.kt:322)
	at kotlinx.coroutines.intrinsics.CancellableKt.startCoroutineCancellable(Cancellable.kt:30)
	at kotlinx.coroutines.intrinsics.CancellableKt.startCoroutineCancellable$default(Cancellable.kt:25)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:110)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:126)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:56)
	at kotlinx.coroutines.BuildersKt.launch(Unknown Source)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch$default(Builders.common.kt:47)
	at kotlinx.coroutines.BuildersKt.launch$default(Unknown Source)
	at com.intellij.vcs.commit.NonModalCommitWorkflowHandler.doExecuteDefault(NonModalCommitWorkflowHandler.kt:160)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler$executeDefault$$inlined$run$lambda$1$1.invoke(AbstractCommitWorkflowHandler.kt:130)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler$executeDefault$$inlined$run$lambda$1$1.invoke(AbstractCommitWorkflowHandler.kt:50)
	at com.intellij.vcs.commit.AbstractCommitWorkflow.continueExecution$intellij_platform_vcs_impl(AbstractCommitWorkflow.kt:137)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler$executeDefault$$inlined$run$lambda$1.invoke(AbstractCommitWorkflowHandler.kt:125)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler$executeDefault$$inlined$run$lambda$1.invoke(AbstractCommitWorkflowHandler.kt:50)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler$refreshChanges$1.run(AbstractCommitWorkflowHandler.kt:206)
	at com.intellij.openapi.vcs.changes.InvokeAfterUpdateCallback$CallbackBase.invokeCallback(InvokeAfterUpdateCallback.java:66)
	at com.intellij.openapi.vcs.changes.InvokeAfterUpdateCallback$ProgressCallback$ModalWaiter.onFinished(InvokeAfterUpdateCallback.java:165)
	at com.intellij.openapi.progress.impl.CoreProgressManager.finishTask(CoreProgressManager.java:605)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.finishTask(ProgressManagerImpl.java:167)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressSynchronously$9(CoreProgressManager.java:563)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:437)
	at com.intellij.openapi.application.ex.ApplicationUtil.invokeAndWaitSomewhere(ApplicationUtil.java:145)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:561)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcessWithProgressSynchronously(ProgressManagerImpl.java:91)
	at com.intellij.openapi.progress.impl.CoreProgressManager.run(CoreProgressManager.java:384)
	at com.intellij.openapi.progress.Task.queue(Task.java:124)
	at com.intellij.openapi.vcs.changes.InvokeAfterUpdateCallback$ProgressCallback.startProgress(InvokeAfterUpdateCallback.java:127)
	at com.intellij.openapi.vcs.changes.UpdateRequestsQueue.invokeAfterUpdate(UpdateRequestsQueue.java:169)
	at com.intellij.openapi.vcs.changes.ChangeListManagerImpl.invokeAfterUpdate(ChangeListManagerImpl.java:368)
	at com.intellij.openapi.vcs.changes.ChangeListManager.invokeAfterUpdateWithModal(ChangeListManager.java:63)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler.refreshChanges(AbstractCommitWorkflowHandler.kt:204)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler.executeDefault(AbstractCommitWorkflowHandler.kt:124)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler.access$executeDefault(AbstractCommitWorkflowHandler.kt:50)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler$executorCalled$1.invoke(AbstractCommitWorkflowHandler.kt:108)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler$executorCalled$1.invoke(AbstractCommitWorkflowHandler.kt:50)
	at com.intellij.vcs.commit.AbstractCommitWorkflow$startExecution$1.invoke(AbstractCommitWorkflow.kt:130)
	at com.intellij.vcs.commit.AbstractCommitWorkflow$startExecution$1.invoke(AbstractCommitWorkflow.kt:66)
	at com.intellij.vcs.commit.AbstractCommitWorkflow.continueExecution$intellij_platform_vcs_impl(AbstractCommitWorkflow.kt:137)
	at com.intellij.vcs.commit.AbstractCommitWorkflow.startExecution$intellij_platform_vcs_impl(AbstractCommitWorkflow.kt:128)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler.executorCalled(AbstractCommitWorkflowHandler.kt:105)
	at com.intellij.vcs.commit.AbstractCommitWorkflowHandler.execute(AbstractCommitWorkflowHandler.kt:103)
	at com.intellij.openapi.vcs.changes.actions.BaseCommitExecutorAction.actionPerformed(BaseCommitExecutorAction.kt:30)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAwareWithCallbacks$4(ActionUtil.java:244)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:265)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:244)
	at com.intellij.util.ui.JButtonAction$createCustomComponent$1.actionPerformed(JButtonAction.kt:27)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:270)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6654)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3345)
	at java.desktop/java.awt.Component.processEvent(Component.java:6419)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5029)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4861)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2790)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4861)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:891)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:820)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:757)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:447)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:818)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:446)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:805)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:498)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
	```

**Comment by Socolin** [comment]
Hello

Can you provide more information ? What version of Rider are you using ?
I cannot find a way to reproduce this. The method seems to exists

**Comment by Socolin** [comment]
I published a new update, let me know if the 1.10.2 fix the problem.

**Comment by inyutin-maxim** [comment]
> Hello
> 
> Can you provide more information ? What version of Rider are you using ? I cannot find a way to reproduce this. The method seems to exists

```
JetBrains Rider 2021.3.2
Build #RD-213.6461.51, built on December 22, 2021
Licensed to Максим Инютин
You have a perpetual fallback license for this version.
Subscription is active until July 29, 2023.
Runtime version: 11.0.13+7-b1751.21 amd64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
Windows 11 10.0
.NET Framework 4.0.30319.42000
GC: G1 Young Generation, G1 Old Generation
Memory: 1500M
Cores: 6
Registry:
    ide.mac.bigsur.window.with.tabs.enabled=false
    debugger.valueTooltipAutoShowOnSelection=true
    rdclient.asyncActions=false
    ide.new.project.model.index.case.sensitivity=true
    indexing.enable.entity.provider.based.indexing=false

Non-Bundled Plugins:
    testlinker2-rider (2021.3)
    tanvd.grazi (213.6461.19)
    org.sonarlint.idea (6.3.1.40498)
    org.example.githubpang.rossynt (213.9.0)
    org.bigtows.PinNote (2.1.1)
    idea.plugin.protoeditor (213.6461.28)
    fluentvalidation-rider (2021.3.0)
    fluentassertions-rider (2021.3.0)
    eu.gillissen.rider.usersecrets (1.2.0)
    com.settler.rider.specflow (1.10.0)
    net.allape.XFTP (0.10.2)
    com.intellij.rider.plugins.nuke (2021.3.0)
    com.intellij.resharper.StructuredLogging (2021.3.0.177)
    com.intellij.resharper.HeapAllocationsViewer (2021.3.0)
    com.gmike.visual-studio-2019-dark (1.5.3)
    com.github.gitduckhq.intellij-plugin (1.1.27)
    com.intellij.plugin.adernov.powershell (2.0.9)
    PlantUML integration (5.9.1)
    Keymap exporter (2.2)
    Key Promoter X (2021.3)
    izhangzhihao.rainbow.brackets (6.21)
    cn.yiiguxing.plugin.translate (3.2.1-213)
    zielu.gittoolbox (212.8.4)
    mobi.hsz.idea.gitignore (4.3.0)
    com.microsoft.vso.idea (1.162.0)
    com.mallowigi (50.2.0)
    com.almightyalpaca.intellij.plugins.discord (1.8.0)
    com.intellij.kubernetes (213.6461.19)
    ru.adelf.idea.dotenv (2021.3.0.213)
    CyclomaticComplexity (2021.3.0)
    Batch Scripts Support (1.0.13)
    Abc.MoqComplete.Rider (1.4.11)

```
